### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by RevathyRNair**
WIP
Kindly review my code. Project details given in README file.

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,174 @@
+@font-face {
+    font-family: "digital-7"; 
+    src: url("https://www.1001fonts.com/digital-7-font.html") format ("TrueType (ttf)");
+  }
+
+body {
```
Comment (abanseka) [inline css/index.css]: - [x] Use a class name here. instead of `body`

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    width: 100vw;
+    height: 100vh;
+    background-image: url(../images/basketball.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    /* display: block; */
```
Comment (abanseka) [inline css/index.css]: - [x] Remove commented code

File: css/index.css [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+}
+
+.display-msg {
+    color: white;
+    font-weight: bold;
+    font-size: 50px;
+    margin-left: 20px;
+    margin-top: 20px;
```
Comment (abanseka) [inline css/index.css]: - [x] Re-evaluate the purpose of this margins.

> am not sure how you want to style this element, but the margin changes nothing, trying removing and see, the msg stays in the same position

File: css/index.css [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    margin: auto;
+    width: 20%;
+    padding: 10px;
+    /* justify-content: center; */
+    
+    
+    /* font-size: large;
+    border: 2px solid;
+    margin-left: 40%;
+    margin-right: 40%; */
```
Comment (abanseka) [inline css/index.css]: - [x] Remove commented code

File: css/index.css [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+    text-align: center;
+    justify-content: center;
+}
+
+.home-score, .guest-score {
+    background-color: #da454549;
+    justify-content: center;
+    color: white;
+    margin-bottom: auto;
+}
```
Comment (abanseka) [inline css/index.css]: - [x] You need a display (`flexbox` or `grid`) to use `justify-content`
- [x] the color white here is not active ie text is still black

File: css/index.css [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+    margin: 5px 0px;
+    /* text-align: center; */
+    
+    
+}
+.score {
+    display: grid;
+    grid-template-columns: 90% 10%;
+    border: 10px solid wheat;
+}
```
Comment (abanseka) [inline css/index.css]: - [x] Remove all of this

File: css/index.css [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+
+.point-board {
+    min-height: 350px;
+    text-align: center;
```
Comment (abanseka) [inline css/index.css]: - [x] Remove this

File: css/index.css [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+    margin-top: 30px;
+}
+
+.team-score-label {
+    color: rgb(232, 241, 46);
+    text-shadow: 2px 2px rgba(0, 0, 255, 0.434);
+}
+
+.score {
+    border: 1px solid black;
```
Comment (abanseka) [inline css/index.css]: - [x] Remove this

> some where at the top you applied a border with wheat color

File: css/index.css [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+    background-color: black;
+}
+
+.home-pt, .guest-pt {
+    font-family: "digital-7";
+    color: rgb(186, 35, 60);
+    font-size: 50px;
+}
+
+.pt-btn {
```
Comment (abanseka) [inline css/index.css]: - [x] use explicit class names (avoid short-hand)

File: css/index.css [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+    background-color: rgb(16, 16, 39);
+    border-radius: 10px;
+    color: white;
+    padding: 5px;
+    font-size: 20px;
+    cursor: pointer;
+}
+#start-btn {
+    justify-content: center;
+    margin: 5% 20% 5% 20%;
```
Comment (abanseka) [inline css/index.css]: - [x] use `margin: (top and bottom value) (left and right value)` | `margin: 5% 20%`

File: index.js [hunk 11]
```diff
@@ -0,0 +1,116 @@
+let rounds = 1;
+let homeTotalScore = 0;
+let guestTotalScore = 0;
+let matchDuration = 12;
```
Comment (abanseka) [inline index.js]: - [x] These should come after all selected elements ie  after `line 26`

File: index.js [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+const homeRound1Score = document.querySelector(".home-round-1");
+const homeRound2Score = document.querySelector(".home-round-2");
+const homeRound3Score = document.querySelector(".home-round-3");
+const homeRound4Score = document.querySelector(".home-round-4");
+
+const homePoint = document.querySelector(".home-pt");
+const guestPoint = document.querySelector(".guest-pt");
+
+const addPointButton = document.querySelectorAll(".add-pt-btn");
+console.log(addPointButton);
```
Comment (abanseka) [inline index.js]: - [x] Remove console logs

**Review by abanseka — CHANGES_REQUESTED** [review]

**Comment by abanseka** [comment]
### General  Review

- [x] start button functional when timer is running
>  while the timer is running, if the start button is clicked again, the timer glitches, showing the newly triggered time and the current running time at once. 
> #### Solution: Disable start button when timer starts
**_[RRN]====Resolved====_**

- [ ] Resetting points on the score board.
> after round is over, even after score points show zero, if  a point is added, it shows the sum of the previous round plus newly added point.

> e.g
> -  Round one runs through, home get's 2 points and guest 1 point, 
> - Round ends points reset showing 0 for both teams
> - Click on +2 for home and it shows 4 and for +2 for guest shows 3 instead of 2 and 1

**_[RRN]====Not able to resolve====_**

- [x] Ensure that css properties achieve a particular goal. if it changes nothing, don't add it.
**_[RRN]====Noted====_**
- [x] Clean up your code before pushing,( Remove commented code and console logs)
**_[RRN]====Noted====_**
- [x] Set your tab spacing  to 2 and ensure that you have prettier working properly.
**_[RRN]====Changed====_**


# Kudos :clinking_glasses:

File: css/index.css (hunk 2 above)
Comment (RevathyRNair) [reply css/index.css]: removed

File: css/index.css (hunk 4 above)
Comment (RevathyRNair) [reply css/index.css]: removed

File: css/index.css (hunk 3 above)
Comment (RevathyRNair) [reply css/index.css]: Noted. removed the margins

File: css/index.css (hunk 6 above)
Comment (RevathyRNair) [reply css/index.css]: Removed

File: css/index.css (hunk 7 above)
Comment (RevathyRNair) [reply css/index.css]: removed

File: css/index.css (hunk 9 above)
Comment (RevathyRNair) [reply css/index.css]: changed

File: css/index.css (hunk 10 above)
Comment (RevathyRNair) [reply css/index.css]: changed

File: index.js (hunk 11 above)
Comment (RevathyRNair) [reply index.js]: changed

File: index.js (hunk 12 above)
Comment (RevathyRNair) [reply index.js]: removed

File: css/index.css (hunk 8 above)
Comment (RevathyRNair) [reply css/index.css]: changed

File: css/index.css (hunk 5 above)
Comment (RevathyRNair) [reply css/index.css]: changed for better display

File: css/index.css (hunk 1 above)
Comment (RevathyRNair) [reply css/index.css]: resolved

File: index.js [hunk 13]
```diff
@@ -6,105 +6,175 @@ const homeRound1Score = document.querySelector(".home-round-1");
 const homeRound2Score = document.querySelector(".home-round-2");
 const homeRound3Score = document.querySelector(".home-round-3");
 const homeRound4Score = document.querySelector(".home-round-4");
+const totalHomeScore = document.querySelector(".total-home-score");
 
 const homePoint = document.querySelector(".home-pt");
 const guestPoint = document.querySelector(".guest-pt");
 
-const addPointButton = document.querySelectorAll(".add-pt-btn");
+var addPointButton = document.querySelectorAll(".add-pt-btn");
```
Comment (abanseka) [inline index.js:14]: use `let` here instead

**Review by abanseka — CHANGES_REQUESTED** [review]

**Review by ahamad-ch — APPROVED** [review]

**Pull request merged**